### PR TITLE
Release 2.3.7

### PR DESCRIPTION
**Pull request by rawbee**
## Description of the change

Release for #45 

## Type of change
- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

https://pinwheel.atlassian.net/browse/PP-5477

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by TrekSoft — APPROVED** [review]
cat-6-3

File: app/build.gradle [hunk 1]
```diff
@@ -17,7 +17,7 @@ android {
         minSdkVersion 22
         targetSdkVersion 30
         versionCode 12
```
Comment (TrekSoft) [inline app/build.gradle:19]: This needs to be bumped

File: pinwheel-android/build.gradle [hunk 2]
```diff
@@ -4,7 +4,7 @@ plugins {
     id 'maven-publish'
 }
 
-def libraryVersion = '2.3.6'
+def libraryVersion = '2.3.7'
```
Comment (TrekSoft) [inline pinwheel-android/build.gradle:7]: Can't add a comment there but on line 23 versionCode needs to be bumped

**Review by TrekSoft — CHANGES_REQUESTED** [review]
Caught a couple versionCode updates that need to be bumped

File: pinwheel-android/build.gradle (hunk 2 above)
Comment (rawbee) [reply pinwheel-android/build.gradle:7]: good catch, thank you!

**Review by TrekSoft — APPROVED** [review]

**Pull request merged**